src/cloud-api-adaptor/test/e2e/common_suite.go

```diff
@@ -615,3 +615,45 @@ func DoTestKbsKeyReleaseForFailure(t *testing.T, e env.Environment, assert Cloud
 
 	NewTestCase(t, e, "DoTestKbsKeyReleaseForFailure", assert, "Kbs key release is failed").WithPod(pod).WithTestCommands(testCommands).Run()
 }
+
+func DoTestRestrictivePolicyBlocksExec(t *testing.T, e env.Environment, assert CloudAssert) {
+	allowAllExceptExecPolicyFilePath := "../../podvm/files/etc/kata-opa/allow-all-except-exec-process.rego"
```

Comment:
Is it possible to avoid relative path and use absolute path here ?

Reply:
Sorry, I'm not sure I understand what the suggestion is here - the rego files are outside of the tests directory, so how do we run out of it without using the relative path?

Reply:
I'm bit uncomfortable using relative paths outside of the e2e dir. This creates hard coupling between tests and what goes in the podvm image. If we change the location of the files, then the test breaks and it puts a cognitive burden to remember changing the paths everywhere.
So how about keeping the test policies under the e2e test directory (say under test-artifacts or any name that you prefer) and reference it from there.
It's similar to the discussion on versions.yaml for tests.

Reply:
Ok, I understand now - so you are proposing duplicating the files